src/util/openfaas/index.js

```diff
@@ -64,6 +66,32 @@ const callWithRetry = async (
   }
 };
 
+const reconcileFunction = async (fnName) => {
+  logger.debug(`Reconciling faas function: ${fnName}`);
+
+  try {
+    const fns = await getFunctionList();
+
+    const lookup = find(fns, (entry) => entry.name === fnName);
+    if (isNil(lookup)) {
+      throw new RespStatusError(404, `${fnName} not found in the list of system functions`);
+    }
+    // versionId and libraryVersionIds are used in the process
+    // to create the envProcess which will be copied from the original
+    // in next step
+    const payload = buildOpenfaasFn(fnName, null, '', '', false, {
+      transformationId: lookup['labels']['transformationId'],
```

Comment:
```suggestion
      transformationId: lookup?.labels?.transformationId,
```
eslint suggests dot notation